tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -44,10 +45,33 @@
 EXAMPLES_KEY = 'examples'
 
 
-def _PartitionFn(record: bytes, num_partitions: int, buckets: List[int]) -> int:
+def _PartitionFn(record: bytes, num_partitions: int, buckets: List[int], feature_name: Text) -> int:
```

Comment:
feature_name pytype is Optionp[Text] as it can be None? (you can find examples in this repo)